classes.py

```diff
@@ -0,0 +1,416 @@
+'''
+THE PYTHON CLASS LIBRARIES for the MealPlan
+
+
+Note
+
+
+
+
+Contributors:
+1. Olusayo Obajemu
+2. ....
+
+CivicTech Fredericton
+February 2020
+
+'''
+
+######################################## CLASS USER ##################################
+class User:
+""" 
+The User Class 001
+
+preliminary notes:
+
+1. Every user has userID and log status
+2. When a user is authorised, the log status is changed to True
+
+"""
+
+	def __init__(self, LoggedIn_ = False, UserID_ = 0, *args, **kwargs):
+		""" 
+		Constructor for initialising the User Object
+		Constructor initialises the 	loggedIn status, 	initial value = False
+										UserID, 			initial value = 0
+
+
+		**kwargs: Additional keyword arguments to pass to a superclass constructor 
+		"""
+
+		self.LoggedIn = LoggedIn_
```

Comment:
This field will be handled through Cognito; if an authenticated session is returned they are logged in.